api/controllers/posts.js

```diff
@@ -1,17 +1,19 @@
-const Post = require("../models/post");
-const TokenGenerator = require("../models/token_generator");
+const Post = require('../models/post');
+const TokenGenerator = require('../models/token_generator');
 
 const PostsController = {
   Index: (req, res) => {
-    Post.find(async (err, posts) => {
-      if (err) {
-        throw err;
-      }
+    Post.find({})
+      .populate('user_id')
```

Comment:
Looked this populate method up, seems extremely helpful for what we are about to do with comments front end! One worth sharing with the group I think :)